examples/vrf/src/handlers/arbiter.rs

```diff
@@ -67,22 +53,21 @@ impl<E: Clock + Spawner, C: Scheme> Arbiter<E, C> {
         let timeout = start + 4 * self.dkg_phase_timeout; // start -> commitment/share -> ack -> arbiter
 
         // Send round start message to contributors
-        let mut group = None;
         if let Some(previous) = &previous {
-            group = Some(previous.serialize());
-            let public = poly::public(previous).serialize();
-            info!(round, public = hex(&public), "starting reshare");
+            info!(round, public=?previous, "starting reshare");
```

Comment:
I found it useful to display the `public` here instead of the full `poly` (as that changes).